### ISSUE TITLE
Problem with Xcode 15

### ISSUE THREAD

**Issue by grimAgent** [lupidan/apple-signin-unity#179]
Looks like since installing the latest Xcode 15 this is no longer working as expected. 
I'm using the `OnPostProcessBuild` script shared in the Readme file and it was working fine until I upgraded. Now i'm getting this error on compile:
```
Error building Player: System.Exception: Can't Add Sign In With Apple programatically in this Unity version
  at AppleAuth.Editor.ProjectCapabilityManagerExtension.AddSignInWithAppleWithCompatibility (UnityEditor.iOS.Xcode.ProjectCapabilityManager manager, System.String unityFrameworkTargetGuid) [0x000ce] in ./Library/PackageCache/com.lupidan.apple-signin-unity@e39981bca0/AppleAuth/Editor/ProjectCapabilityManagerExtension.cs:40 
  at SignInWithApplePostprocessor.OnPostProcessBuild (UnityEditor.BuildTarget target, System.String path) [0x00059] in /Users/jan/Projects/unity/Abysswalkers/Assets/Editor/SignInWithApplePostprocessor.cs:29 
```

Unity version is 2022.3.10f1 LTS

**Comment by grimAgent** [comment]
Looks like someone made a fix for this here? https://github.com/lupidan/apple-signin-unity/pull/178

**Comment by grimAgent** [comment]
For what is worth, I tried the fix on that pull request and it fixes the issue. So if possible, it would be great to accept that PR

**Comment by lupidan** [comment]
Fixed in v1.4.3

**Comment by derwaldgeist** [comment]
Thanks!
